pkgs/tools/audio/beets/default.nix

```diff
@@ -40,36 +41,40 @@ let
     })
   ];
 in
-lib.makeExtensible (self: {
-  beets = self.beets-stable;
+lib.makeExtensible (
+  self:
+  {
+    beets = self.beets-stable;
 
-  beets-stable = callPackage ./common.nix rec {
-    inherit python3Packages extraPatches;
-    version = "2.2.0";
-    src = fetchFromGitHub {
-      owner = "beetbox";
-      repo = "beets";
-      tag = "v${version}";
-      hash = "sha256-jhwXRgUUQJgQ/PLwvY1UfHCJ9UC8DcdBpE/janao0RM=";
+    beets-stable = callPackage ./common.nix rec {
+      inherit python3Packages extraPatches;
+      version = "2.2.0";
+      src = fetchFromGitHub {
+        owner = "beetbox";
+        repo = "beets";
+        tag = "v${version}";
+        hash = "sha256-jhwXRgUUQJgQ/PLwvY1UfHCJ9UC8DcdBpE/janao0RM=";
+      };
     };
-  };
 
-  beets-minimal = self.beets.override { disableAllPlugins = true; };
+    beets-minimal = self.beets.override { disableAllPlugins = true; };
 
-  beets-unstable = callPackage ./common.nix {
-    inherit python3Packages;
-    version = "2.2.0-unstable-2025-03-12";
-    src = fetchFromGitHub {
-      owner = "beetbox";
-      repo = "beets";
-      rev = "670a3bcd17a46883c71cf07dd313fcd0dff4be9d";
-      hash = "sha256-hSY7FhpPL4poOY1/gqk7oLNgQ7KA/MJqx50xOLIP0QA=";
+    beets-unstable = callPackage ./common.nix {
+      inherit python3Packages;
+      version = "2.2.0-unstable-2025-03-12";
+      src = fetchFromGitHub {
+        owner = "beetbox";
+        repo = "beets";
+        rev = "670a3bcd17a46883c71cf07dd313fcd0dff4be9d";
+        hash = "sha256-hSY7FhpPL4poOY1/gqk7oLNgQ7KA/MJqx50xOLIP0QA=";
+      };
     };
-  };
 
-  alternatives = callPackage ./plugins/alternatives.nix { beets = self.beets-minimal; };
-  audible = callPackage ./plugins/audible.nix { beets = self.beets-minimal; };
-  copyartifacts = callPackage ./plugins/copyartifacts.nix { beets = self.beets-minimal; };
-
-  extrafiles = throw "extrafiles is unmaintained since 2020 and broken since beets 2.0.0";
-})
+    alternatives = callPackage ./plugins/alternatives.nix { beets = self.beets-minimal; };
+    audible = callPackage ./plugins/audible.nix { beets = self.beets-minimal; };
+    copyartifacts = callPackage ./plugins/copyartifacts.nix { beets = self.beets-minimal; };
+  }
+  // lib.optionalAttrs config.allowAliases {
+    extrafiles = throw "extrafiles is unmaintained since 2020 and broken since beets 2.0.0";
```

Comment:
Could you explain a bit the reasoning behind this change? Are you saying that we should remove completely such old `throw` lines, depending on whether aliases are enabled or not? I'm mostly perplexed why did you bother making such a minor change and sending a PR for it.